sponsorship.md

```diff
@@ -10,12 +10,24 @@ As a free-to-attend gathering, our attendees don't have to worry about huge conf
 
 In exchange for financial backing, we would provide:
```

Comment:
In exchange for financial backing, we are offering supporters a selection of sponsorship packages to choose from.

Basic sponsorship package
This includes:
- blah blah